src/alpine/3.6-ForHelix/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM alpine:3.6
+
+RUN apk update
+
+RUN apk add --no-cache \
+        autoconf \
+        bash \
+        build-base \
+        clang \
+        clang-dev \
+        cmake \
+        coreutils \
+        curl-dev \
+        gcc \
+        gettext-dev \
+        git \
+        icu-dev \
+        krb5-dev \
+        libtool \
+        libunwind-dev \
+        libffi \
+        linux-headers \
+        llvm \
+        make \
+        openssl \
+        openssl-dev \
+        paxctl \
+        py-cffi \
+        python \
+        python-dev \
+        sudo \
+        util-linux-dev \
+        zlib-dev
+
+RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
+        userspace-rcu-dev \
+        lttng-ust-dev
+
+RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/testing add --no-cache \
+        lldb-dev
+
+RUN /usr/sbin/adduser -D -G adm -s /bin/sh helixbot
```

Comment:
Do you want the containers to run as this user by default?  Curious if you want to specify the `USER helixbot` instruction.

Reply:
I see, we could do that definitely but for now I've been creating this user and then having the run command set `-u helixbot` as part of it. What is the recommendation here?

Reply:
Since the resulting image is really intended to always run as the helixbot, it feels like it would be better to encapsulate it here.  As I noted later on, there are some advantages within the Dockerfile as well of using the USER instruction.

Reply:
+1.  We originally thought this would be configurable but helix clients always running as helixbot is good.

Reply:
this can't be configurable. This dockerfile is a reciepe that produces an image, so once the image is produced, you can't change what comes with it until you rebuild a new image. That means that if we decide to change the username, we would have to rebuild this dockerfile anyway.